pkg/antctl/command_definition.go

```diff
@@ -46,6 +46,10 @@ const (
 
 const (
 	sortByEffectivePriority string = "effectivePriority"
+	sortBySourceName        string = "{.sourceRef.name}"
+	sortByCreationTimestamp string = "{.metadata.creationTimestamp}"
+	sortByUID               string = "{.metadata.uid}"
+	sortByName              string = "{.metadata.name}"
```

Comment:
there should be no need to hardcode this. Any valid field path should be supported.